apim/3.x/values.yaml

```diff
@@ -764,7 +764,7 @@ gateway:
       #     nginx.ingress.kubernetes.io/enable-rewrite-log: "true"
       #     kubernetes.io/app-root: /gateway
       #     kubernetes.io/rewrite-target: /gateway
-      #     nginx.ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\nproxy_set_header if-match \"\";\n"
+      #     nginx.ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\nproxy_pass_header if-match;\n"
```

Comment:
@ytvnr same here, I think it's useless here.

Reply:
done